backend/.env-example

```diff
@@ -59,7 +58,7 @@ FRONTEND_URL="http://localhost:3000"
 # TWILIO_API_KEY=""
 # TWILIO_ACCOUNT_SID=""
 
-DD_ENV="local"
-DD_SERVICE="postman"
-DD_LOGS_INJECTION="true"
-DD_TRACE_ENABLED="false"
\ No newline at end of file
+# DD_ENV="local"
```

Comment:
Are these intentionally commented out?

Reply:
Yup I was cleaning up the env variables in `backend/.env-example` such that optional env vars are commented out but compulsory env vars are not (so someone setting it up from scratch will know which is which)

which reminds me, I forgot to add the optional `USD_TO_SGD_RATE` env variable

Reply:
added! https://github.com/opengovsg/postmangovsg/pull/1582/commits/3ed55112b30ef9a39e9a1ce2618f8965de5acc83